kivy/core/window/__init__.py

```diff
@@ -2032,17 +2032,25 @@ def on_textinput(self, text):
         '''
         pass
 
-    def on_drop_begin(self, x, y):
+    def on_drop_begin(self, x, y, *args):
         '''Event called when a text or a file drop on the application is about
         to begin. It will be followed-up by a single or a multiple
         `on_drop_text` or `on_drop_file` events ending with an `on_drop_end`
         event.
 
+        Arguments `x` and `y` are the mouse cursor position at the time of the
+        drop and you should only rely on them if the drop originated from the
+        mouse.
+
         :Parameters:
             `x`: `int`
-                Cursor x position, relative to window :attr:`left`.
+                Cursor x position, relative to the window :attr:`left`, at the
+                time of the drop.
             `y`: `int`
-                Cursor y position, relative to window :attr:`top`.
+                Cursor y position, relative to the window :attr:`top`, at the
```

Comment:
Is it really relative to the top? E.g. the bottom left corner is `-h`?

Reply:
It's the same origin (0, 0) position as for the `on_mouse_xxx` events, which is the window left/top.

Reply:
Ok, didn't remember that that was the origin.

Reply:
It is and imho it should be changed to left/bottom be inline with the touch/motion events, but that would be a breaking change.